src/main/java/com/backbase/ct/bbfuel/input/TransactionsReader.java

```diff
@@ -24,7 +24,8 @@ public TransactionsPostRequestBody loadSingle(String externalArrangementId) {
         return getRandomFromList(load(globalProperties.getString(CommonConstants.PROPERTY_TRANSACTIONS_DATA_JSON)))
                 .withId(UUID.randomUUID().toString())
                 .withArrangementId(externalArrangementId)
-                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)));
+                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)))
+                .withCheckSerialNumber(generateRandomNumberInRange(1, 100));
```

Comment:
Can you make it in the range of [0-9999999999999999]
checkSerialNumber": {
      "type": "integer",
      "minimum": 0,
      "maximum": 9999999999999999,
      "description": "Serial number of the check. Each check has a different number for identification purposes."